### DISCUSSION TITLE
Agent Swarm Support

### DISCUSSION THREAD

**Discussion by orkutmuratyilmaz** [safevideo/autollm#151] [Feature Requests]
Hello and thanks for your great tool,

"Agent Swarm" concept is getting high ratings now and it would be nice to have some documentation for creating an "Agent Swarm" with AutoLLM.

A video example [here](https://www.youtube.com/watch?v=geLX30qax8Q) and the notebook [here](https://colab.research.google.com/drive/1Tc-nD32hE7qB1a4Evjcw1NJUrDbc7H0Z?usp=sharing).

Best,
Orkut

**Comment by fcakyon** [comment]
That is on our roadmap! Will publish several releases in the following weeks which will eventually let our users to combine different llm agents 💯

**Reply by orkutmuratyilmaz** [reply]
Thanks for the fast reply and good news too:)
